server/views/pages/pdfHeader.njk

```diff
@@ -1,38 +1,55 @@
-<html>
+<html lang="en">
 <head>
-    <style>
-        body {
-            -webkit-print-color-adjust: exact;
-            font-family: arial;
-        }
-        h1 {
-            font-size: 18px;
-        }
-        h2 {
-            font-size: 10px;
-            font-weight: normal;
-        }
-        .adjudication-pdf-header {
-            background-color: #d8d8d8;
-            width: 100%;
-            display: flex;
-            justify-content: space-between;
-            align-items: center;
-            padding-left:30px;
-            padding-right: 30px;
-        }
-        .adjudication-pdf-header-bold {
-            font-weight: bold;
-        }
-        .adjudication-pdf-header-normal {
-            font-weight: normal;
-        }
-    </style>
+  <style>
+      body {
+          font-family: Arial, sans-serif;
+      }
+      .dps-logo {
+          float: left;
+          margin-left: 20px;
+          margin-right: 20px;
+          padding-bottom: 20px;
+      }
+      h3 {
+          margin-top: 0;
+          padding-top: 0;
+      }
+      .notice {
+          margin-right: 10px;
+          font-size: 12px;
+      }
+  </style>
+  <title>Unused</title>
 </head>
 <body>
-    <div class='adjudication-pdf-header'>
-        <h1>Conduct report</h1>
-        <h2><span class='adjudication-pdf-header-bold'>Charge number:</span> <span class="adjudication-pdf-header-normal">{{ chargeNumber }}</span></h2>
-    </div>
+<div>
+  <div class='dps-logo'>
+    <svg width="100" height="100" viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
copied from `dps-logo.svg` in DPS.  Needs to be included here separately as gotenberg headers can't read images